guides/source/active_record_basics.md

```diff
@@ -45,6 +45,8 @@ relationships of the objects in an application can be easily stored and
 retrieved from a database without writing SQL statements directly and with less
 overall database access code.
 
+NOTE: If you are not familiar with relational database management systems (rDBMS), but want to know more, please do this tutorial.
```

Comment:
I see what you're saying about not understanding rDBMS but I don't know of a tutorial to link here so let's remove this for now.

Reply:
I can try to find some tutorials myself, but my question is what should I pick?

SQL in general? ([Wikibook](https://upload.wikimedia.org/wikipedia/commons/6/6d/Structured_Query_Language.pdf) explains basics and that's where I learned it).

Or SQLite? As I understand Rails uses this implementation and you even have [a link to a site](https://www.sqlite.org/), but there is only a "Getting started" guide there and it's just awful for those who are new to SQL.

Or MySQL? Since it's used in Rails guides. Btw, can you answer why?